meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-bsp/u-boot/u-boot-ti-staging_%.bbappend

```diff
@@ -17,12 +18,22 @@ SRC_URI:append:am62xx-evm = " \
     file://lmp.cfg \
 "
 
+PATCHTOOL = "git"
+
 PACKAGECONFIG[optee] = "TEE=${STAGING_DIR_HOST}${nonarch_base_libdir}/firmware/tee-pager_v2.bin,,optee-os-fio"
 
+
 # setting DEPENDS create dependency loops so skip the check
 LMPSTAGING_DEPLOYED_CHECK_SKIP += "${PN}:do_deploy"
 
 python() {
     # we need to set the DEPENDS as well to produce valid SPDX documents
     fix_deployed_depends('do_install', d)
+
+    if os.path.exists("${TOPDIR}/conf/keys/custMpk.pem"):
+        d.setVar('K3_ROT_KEYS', 'CONFIG_SIGN_KEY_PATH=${TOPDIR}/conf/keys')
+    else:
+        d.setVar('K3_ROT_KEYS', '')
 }
+
```

Comment:
We can consider the lmp keys the default ones and if the user whats to use the TI keys they just need to unset it with `K3_ROT_KEYS=""` in another bbappend
```
K3_ROT_KEYS ?= "CONFIG_SIGN_KEY_PATH=${TOPDIR}/conf/keys"

EXTRA_OEMAKE += "$K3_ROT_KEYS"
```

Reply:
With that we don't need the python snip to setVar

Reply:
It is not so much saving a python snip but the logic that I disagree with.

IMO the default keys should be the u-boot keys in the project.  And by default I mean them being available without having to specify any options (set or unset)

And then, if the user prefers to use their own keys, then let them figure out how by finding out the variable they need to set.

Preemptively asking them to unset a variable is a time waster IMO.

Reply:
I also think it's preferable for the process to be aligned with all the other board and because to build meta-lmp we have a strict dependency on the lmp-manifest the keys will always be there.
To use the key available on the tree the user can set `CONFIG_SIGN_KEY_PATH="../keys"` or don't use at all the `CONFIG_SIGN_KEY_PATH` on the recipe.